wgpu-types/src/features.rs

```diff
@@ -1193,6 +1193,17 @@ bitflags_array! {
         ///
         /// This is a native only feature.
         const EXPERIMENTAL_MESH_SHADER_MULTIVIEW = 1 << 49;
+
+        /// Ensures that `dot4I8Packed` and `dot4U8Packed` in WGSL are compiled to specialized
+        /// intrinsics.
+        ///
+        /// Supported platforms:
+        /// - Vulkan (with SPIR-V 1.6+ and [DotProductInput4x8BitPackedKHR](https://registry.khronos.org/vulkan/specs/latest/html/vkspec.html#spirvenv-capabilities-table-DotProductInput4x8BitPacked))
+        /// - DX12 (with SM 6.4+)
+        ///
+        /// Potential Platforms:
+        /// - WebGPU
+        const NATIVE_PACKED_INTEGER_DOT_PRODUCT = 1 << 50;
```

Comment:
I don't think this should be a new feature, users should not have to enable it.

Reply:
Removed it.